src/nvim/mark_extended_defs.h

```diff
@@ -10,6 +10,7 @@ typedef struct {
 } VirtTextChunk;
 
 typedef kvec_t(VirtTextChunk) VirtText;
+#define VIRTTEXT_EMPTY ((VirtText)KV_INITIAL_VALUE)
```

Comment:
do we normally name this `VIRTTEXT_INIT` ? (cf. `STRING_INIT` et al.)

Reply:
no, this is used only for _value_ locations where virt_text is optional, and not as _variable_ initialization before starting to use the array (use KV_INITIAL_VALUE for that). variable `_INIT` defines like `STRING_INIT` does not have a type specifier like this.